### DISCUSSION TITLE
如何从默认的 sqlite3 改成 MySQL？

### DISCUSSION THREAD

**Discussion by kilvn** [AlistGo/alist#1705] [Invalid question]
使用的Linux一键安装脚本 装在 ubuntu22里面，自动装完后自动创建config文件，存储使用的 sqlite3，想改成MySQL存储，没有看到任何指引文档和讨论，请问怎么迁移呢？

**Comment by anwen-anyi** [comment]
https://www.bilibili.com/video/BV1iV4y1T7kh

**Reply by kilvn** [reply]
可以可以

**Reply by tpfortree** [reply]
这么Hacker的方式修改，，， 有点惊悚~~~
